src/BigANNBinaryEmbeddingScenarioBase.cs

```diff
@@ -63,10 +63,12 @@ protected string GetQueryDataPath()
 
         protected override ContainerProperties GetContainerSpec(string containerName)
         {
+            var quantizationByteSize = Convert.ToInt32(this.Configurations["AppSettings:scenario:quantizationByteSize"]);
```

Comment:
Make it so that if we specify value in appsetting the code uses it else it skips providing an explicit default so serverside can pick default based on table.